components/byu-faculty-profile/byu-faculty-profile-common.scss

```diff
@@ -39,7 +42,7 @@
 }
 
 @mixin cardContent() {
-  padding: 0px 16px 16px 16px;
+  padding: 8px 19px 16px 19px;
```

Comment:
What was the idea behind increasing the padding by 3px? I wasn't able to find it in the inspector. If we can leave it 16px that would be nice because it matches with the material design spec.